package.json

```diff
@@ -397,7 +397,7 @@
                 },
                 "gitlens.explorers.commitFormat": {
                     "type": "string",
-                    "default": "${message} \u00a0\u2022\u00a0 ${authorAgoOrDate} \u00a0 (${id})",
+                    "default": "${message}  •  ${authorAgoOrDate}   (${id})",
```

Comment:
NPM did all those changes automatically with `npm install`. I can revert them if requested. But it should happen to everyone doing `npm install` ?

Reply:
Yes, please revert any changes that aren't absolutely required. Sadly the `package.json` rewriting by npm is a constant source of pain (but if the changes are allowed it screws up the encoding of the file and causes other issues across platforms). I typically just run `npm install --no-save` to avoid that rewriting, but obviously you can't if you are adding/updating a package. Its a total PITA, sorry.